lua/completion/source/lsp.lua

```diff
@@ -30,15 +30,17 @@ local function get_completion_word(item, prefix, suffix)
       else
           newText = item.textEdit.newText
       end
-    if protocol.InsertTextFormat[item.insertTextFormat] == "PlainText"
-		or opt.get_option('enable_snippet') == "snippets.nvim" then
+    if item.insertTextFormat == nil
```

Comment:
Better coding style for this kind of operations:
```suggestion
    if not item.insertTextFormat
```

Reply:
Thank you.